proto/proto-backplane-grpc/src/main/proto/deephaven/proto/object.proto

```diff
@@ -24,3 +84,43 @@ message FetchObjectResponse {
     bytes data = 2;
     repeated io.deephaven.proto.backplane.grpc.TypedTicket typed_export_id = 3;
 }
+
+/*
+ * First payload to send on a MessageStream, indicating the object to connect to
+ * on the server.
+ */
+message ConnectRequest {
+    io.deephaven.proto.backplane.grpc.TypedTicket typed_ticket = 1;
+}
+
+/*
+ * A generic payload that can be sent by either the server or the client, containing
+ * arbitrary binary data, and any number of typed tickets. The current convention is
+ * that the client can send tickets it was given from the server, and the server will
+ * send server-created export tickets for new objects.
+ */
+message Data {
+    bytes data = 1;
+    repeated io.deephaven.proto.backplane.grpc.TypedTicket typed_export_id = 2;
+}
+
+/*
+ * Client payload for the MessageStream.
+ */
+message StreamRequest {
+    oneof payload {
```

Comment:
I'm leaning towards calling this "message", and renaming Data.data to "payload" (plus `source_id` to `connect`), but whatever we pick needs to match the MessageStream.onMessage params too - with the possible exception of "references" vs "typed_export_id"s.